src/tests/java/htsjdk/variant/vcf/VCFHeaderUnitTest.java

```diff
@@ -333,6 +333,14 @@ public void testVCFHeaderQuoteEscaping() throws Exception {
         final VCFSimpleHeaderLine addedHeaderLine = new VCFSimpleHeaderLine("GATKCommandLine.Test", attributes);
         originalHeader.addMetaDataLine(addedHeaderLine);
 
+        final VCFFilterHeaderLine originalCopyAnnotationLine1 = originalHeader.getFilterHeaderLine("ANNOTATION");
+        Assert.assertNotNull(originalCopyAnnotationLine1);
+        Assert.assertEquals(originalCopyAnnotationLine1.getGenericFieldValue("Description"), "ANNOTATION != \"NA\" || ANNOTATION <= 0.01", originalCopyAnnotationLine1.toString());
+
+        final VCFFilterHeaderLine originalCopyAnnotationLine2 = originalHeader.getFilterHeaderLine("ANNOTATION2");
+        Assert.assertNotNull(originalCopyAnnotationLine2);
+        Assert.assertEquals(originalCopyAnnotationLine2.getGenericFieldValue("Description"), "ANNOTATION with quote \" that is unmatched");
```

Comment:
This test case will probably fail if you accept the changes above to handle unclosed quotes.